### DISCUSSION TITLE
do kubernetes monitoring tools qualify for this list?

### DISCUSSION THREAD

**Discussion by rsanford-hh2** [veggiemonk/awesome-docker#1129] [General]
after finding lazydocker via this list i managed to find [k9s](https://k9scli.io/) which is similar for kubernetes. 

i can fork and create a PR to include it but wanted to see if kubernetes specific tooling is appropriate for this list before so doing.

**Comment by mashb1t** [comment]
only if they're docker based and are suitable for this list
